### ISSUE TITLE
How do I compute validation loss during training?

### ISSUE THREAD

**Issue by Linyi-Li** [Megvii-BaseDetection/YOLOX#871]
I want to see the curve of validation loss.
Then I can compare the training and validation loss, to check if the model is overfitting or not.
But the validation loss is not computed in the model.  When I set my model to evaluation --> model.eval(), there is no validation loss returned.
Can anyone help me? Or have good suggestions

**Comment by FateScript** [comment]
What's the meaning of validation loss?  Loss on validation set without bp ?  If so, you couldn't use `model.eval()`. Just do what trainer do without loss.backward and optimizer.step
